### DISCUSSION TITLE
Request for Communication Channel (Slack, etc)

### DISCUSSION THREAD

**Discussion by GlenDsza** [ruxailab/RUXAILAB#310] [General]
Hey Team @marcgc21 , would request y'all to provide a communication channel such as slack workspace or similar, as interested contributors then can work on interested projects and collaborate and discuss about issues. As communicating via github discussion for minor issues could be challenging.

**Comment by marcgc21** [comment]
Yep @GlenDsza! We are working on that but until now this is the best approach to get insights about how to start working and what to do. During the next days we will share this with you!

**Comment by KarinePistili** [comment]
Good news! We just launched our [Discord Server](https://discord.gg/YnkDk9BNYK)
